packages/shared/src/components/fields/PasswordField.tsx

```diff
@@ -45,17 +46,20 @@ export function PasswordField({
     ? `Password need a minimum length of ${props.minLength}`
     : passwordStrengthStates[passwordStrengthLevel].label;
   const hint = !hasUserAction ? null : userActionHint;
-  const shouldShowStrength = showStrength && hasUserAction;
+  const shouldShowStrength = !!value && showStrength && hasUserAction;
```

Comment:
It should always validate on submit though, just making sure that happens (even when input is empty)

Reply:
I think when it is empty, the hint should be different, that it is required/missing.